sql/core/src/test/java/test/org/apache/spark/sql/execution/datasources/xml/JavaXmlSuite.java

```diff
@@ -82,7 +82,7 @@ private Path getEmptyTempDir() throws IOException {
     public void testXmlParser() {
         Map<String, String> options = new HashMap<>();
         options.put("rowTag", booksFileTag);
-        Dataset<Row> df = spark.read().options(options).format("xml").load(booksFile);
+        Dataset<Row> df = spark.read().options(options).xml(booksFile);
```

Comment:
Why change this line?

Reply:
Switched to shortened version for better code readability.

Reply:
Because this change is not related to the theme, it is not recommended to modify it. But it does simplify the code a bit and is also OK.

Reply:
Agreed.